File: javascript/js file/vote check.js
function checkVotingEligibility() {
    let age = prompt("Please enter your age:");
    age = Number(age);
    if (isNaN(age)) {
        console.log("Please enter a valid number for your age.");
        return;
    }
    if (age >= 18) {
        console.log("You are eligible to vote.");
    } else {
        console.log("You are not eligible to vote.");
    }
}
checkVotingEligibility();

File: c++/C++ files/vote check.cpp
#include <iostream>
using namespace std;

int main() {
    const int min_vote_age = 18;
    int age;
    cout << "Enter your age: ";
    cin >> age;
    if (age >= min_vote_age) {
        cout << "You are eligible to vote!" << endl;
    } else {
        cout << "You are not eligible to vote yet." << endl;
    }

    return 0;
}
